pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -542,8 +542,10 @@ type Model struct {
 	Type string `xml:"type,attr"`
 }
 
+// TODO <target dev='eth0' managed='no' </target>
```

Comment:
The purpose of this TODO is illegible to me.

Reply:
Done.